tests/e2e/specs/out-of-game/home.spec.js

```diff
@@ -682,3 +682,11 @@ describe('Announcement Dialogs', () => {
     cy.get('[data-cy=announcement-dialog]').should('not.exist');
   });
 });
+
+// Check for canonical link in head
+describe('Canonical Link', () => {
+  it('Checks for the canonical link in the head meta data on login/signup page', () => {
+    cy.visit('/login');
+    cy.get('head link[rel="canonical"]').should('have.attr', 'href').and('contain', 'https://www.cuttle.cards/signup');
```

Comment:
To wrangle nuance around localhost vs production, let's just make it check for the end of the url without the domain 
```suggestion
    cy.get('head link[rel="canonical"]').should('have.attr', 'href').and('contain', '/signup');
```

Reply:
Hey @itsalaidbacklife I have moved the location of the test and shorted the condition to only the route and not the entire url. [commit](https://github.com/cuttle-cards/cuttle/pull/1145/commits/3d1f075eba5e6e6b19d80f0a9dfb9bc6cbafa3f8)